lib/client/renderer.js

```diff
@@ -472,14 +487,84 @@ function init (client, d3) {
         client.tooltip.transition().duration(TOOLTIP_TRANS_MS).style('opacity', .9)
           .style('left', left + 'px')
           .style('top', (d3.event.pageY ? d3.event.pageY + 15 : 40)  + 'px');
+
+        deleteRect = {
+          x: 0,
+          y: 50,
+          width: 50,
+          height: chart().yScale(chart().yScale.domain()[0]) - 100
+        };
+        chart().drag.append('rect')
+          .attr({
+            class:'drag-droparea',
+            x: deleteRect.x,
+            y: deleteRect.y,
+            width: deleteRect.width,
+            height: deleteRect.height,
+            fill: 'red',
+            opacity: 0.4
+          });
+        if (treatment.insulin) {
+          insulinRect = {
+            x: 0,
+            y: 0,
+            width: chart().charts.attr('width'),
+            height: 50
+          };
+        }
+        chart().drag.append('rect')
+          .attr({
+            class:'drag-droparea',
+            x: insulinRect.x,
+            y: insulinRect.y,
+            width: insulinRect.width,
+            height: insulinRect.height,
+            fill: 'white',
+            opacity: 0.4
+          });
+        if (treatment.carbs) {
+          carbsRect = {
+            x: 0,
+            y: chart().yScale(chart().yScale.domain()[0]) - 50,
+            width: chart().charts.attr('width'),
+            height: 50
+          };
+        }
+        chart().drag.append('rect')
+          .attr({
+            class:'drag-droparea',
+            x: carbsRect.x,
+            y: carbsRect.y,
+            width: carbsRect.width,
+            height: carbsRect.height,
+            fill: '#0099ff',
+            opacity: 0.4
+          });
+          
+        operation = "Move";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1849529815/issues/source?bid=2452379&fileBranchId=3084282#l544)